pkgs/tools/networking/getmail6/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "getmail6";
+  version = "6.9";
+
+  src = fetchFromGitHub {
+    owner = "${pname}";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "09ii5z6nah21y6d49jfqrbqg6y6rvk764jg27yxlq4wlklhb4lmq";
+  };
+
+  doCheck = false;
+
+  postPatch = ''
+    # getmail spends a lot of effort to build an absolute path for
+    # documentation installation; too bad it is counterproductive now
+    sed -e '/datadir or prefix,/d' -i setup.py
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A program for retrieving mail";
+    homepage = "https://getmail6.org";
+    updateWalker = true;
+    license = licenses.gpl2Plus;
+    platforms = platforms.all;
```

Comment:
```suggestion
```
I recently learned that this is not required.

Reply:
Thank you for your suggestion. I have implemented it.